src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -145,13 +146,13 @@ function getMediaSourceHtml(user, item, version) {
         if (stream.BitDepth) {
             attributes.push(createAttribute(globalize.translate('MediaInfoBitDepth'), `${stream.BitDepth} bit`));
         }
-        if (stream.VideoRange) {
+        if (stream.VideoRange && stream.Type === 'Video') {
```

Comment:
Isn't `stream.VideoRange` check enough?

Reply:
When the video stream does not exist (audio only), its default value is `VideoRange.Unknown`. So this makes sense to me.